src/js/components/Form.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import uuidvl from "uuid";
+import { addArticle } from "../actions/index";
+
+const mapDispatchToProps = dispatch => {
+  return {
+    addArticle: article => dispatch(addArticle(article))
+  };
+};
+
+class ConnectedForm extends Component {
```

Comment:
You can use PureComponent for better performance, It's okay to replace all Components in a project into PureComponents, as long as you don't mutate directly any state

Reply:
modified. I didn't know about PureComponent so the concept feels interesting :)